.github/workflows/ci_build_scm_ubuntu_22.04_nvidia.yml

```diff
@@ -1,19 +1,19 @@
-name: CI test to build the CCPP-SCM on ubuntu v22.04
+name: build the CCPP-SCM on ubuntu v22.04
```

Comment:
I thought v20.04 is being sunset, not v22.04?
Anyways, we're using the latest version now. Remove version number from name

Reply:
Do it!